src/crimson/os/seastore/segment_cleaner.h

```diff
@@ -633,12 +633,21 @@ class SegmentCleaner : public SegmentProvider {
   void update_journal_tail_target(journal_seq_t target);
 
   void init_mkfs(journal_seq_t head) {
+    if (head.offset.get_addr_type() == addr_types_t::RANDOM_BLOCK) {
+      journal_tail_target = JOURNAL_SEQ_NULL;
+      journal_tail_committed = JOURNAL_SEQ_NULL;
+      journal_head = JOURNAL_SEQ_NULL;
+      return;
+    }
```

Comment:
SegmentCleaner or whatever it eventually becomes will eventually need to perform journal trimming even if the journal is CBJournal.  This seems wrong.